python/selfie-lib/tests/ArrayMap_test.py

```diff
@@ -0,0 +1,87 @@
+import unittest
+from selfie_lib.ArrayMap import ArrayMap
+
+class ArrayMapTest(unittest.TestCase):
```

Comment:
Pytest can run `unittest`, but all the rest of our tests are in pytest style. GPT4 is pretty good at converting from one to the other.